cmd/kube-controller-manager/app/bootstrap.go

```diff
@@ -20,11 +20,14 @@ import (
 	"context"
 	"fmt"
 
+	"k8s.io/klog/v2"
+
 	"k8s.io/controller-manager/controller"
 	"k8s.io/kubernetes/pkg/controller/bootstrap"
 )
 
 func startBootstrapSignerController(ctx context.Context, controllerContext ControllerContext) (controller.Interface, bool, error) {
+	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "bootstrap-signer"))
```

Comment:
1. Do we care that there is a one character difference `-` from the [controller name](https://github.com/kubernetes/kubernetes/blob/1e3946ce9dcb745771b03845d4afd97caa7c7737/cmd/kube-controller-manager/app/controllermanager.go#L455) `bootstrapsigner`? From consistency perspective, people could be potentially used to the controller name from running the KCM (`--controllers`). Not sure if that is an actual problem though.

2. There is also an inconsistent naming across controllers: just by picking a few
  - deployment
  - cronjob-controller
  - NodeIpamController

We should choose a consistent format/naming across all of the controllers.

Reply:
Consistency is important. If there are already established names, then please point them out as part of your review and use `/lgtm cancel` if necessary to track PRs that aren't ready yet.

Reply:
for the 1. point I think both sides make sense. The logs will get much more exposure and could make sense to make them readable. I will let @soltysh to give his view on this.

2. so in that case I would suggest we use the following format everywhere
- deployment-controller
- cronjob-controller
- nodeipam-controller or node-ipam-controller

Reply:
initiated a conversation in k8s dev: https://groups.google.com/u/1/a/kubernetes.io/g/dev/c/I9l8SSM599s

Reply:
for posterity, implementing aliases in https://github.com/kubernetes/kubernetes/pull/115813